service/src/main/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityValidationService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.rd.model.Organisation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class LocalAuthorityValidationService {
+    private final OrganisationService organisationService;
+    private final FeatureToggleService featureToggleService;
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+    private final LocalAuthorityService localAuthorityNameService;
+
+    public List<String> validateIfUserIsOnboarded() {
+        List<String> errors = new ArrayList<>();
+        String caseLocalAuthority = localAuthorityNameService.getLocalAuthorityCode();
+        String localAuthorityName = localAuthorityNameLookupConfiguration.getLocalAuthorityName(caseLocalAuthority);
+
+        if (featureToggleService.isAllowCaseCreationForUsersNotOnboardedToMOEnabled(localAuthorityName) == false) {
+
+            if (organisationService.findOrganisation().equals(Organisation.builder().build())) {
```

Comment:
tbh that looks a bit weird. Ideally organisationService.findOrganisation() should be changed to return Optional

Reply:
I'm not going to change this method in this PR as it will affect other tests in applicant etc - i've moved 'Organisation.builder().build()' into a static variable at top for more readability